src/profile/components/ProfileCard/style.less

```diff
@@ -41,6 +48,7 @@
 
   .info {
     display: inline-block;
+    max-width: calc(100% - 145px);

```

Comment:
Nitpick but a good convention for accessibility reasons is using [rem length values in CSS](https://www.freecodecamp.org/news/what-is-rem-in-css/). 1rem = 16px so 10rem = 160px and it's usually fine to round to the nearest whole rem rather than converting it exactly. Also wondering if you can explain what the 145px hardcoded value is for

Reply:
Also edited some screenshots into the original comment 👍